core/src/Cuda/Kokkos_Cuda_Instance.hpp

```diff
@@ -39,7 +39,12 @@ struct CudaTraits {
   static constexpr CudaSpace::size_type ConstantMemoryCache =
       0x002000; /*  8k bytes */
   static constexpr CudaSpace::size_type KernelArgumentLimit =
+#if !defined(KOKKOS_COMPILER_NVCC) || defined(KOKKOS_ARCH_MAXWELL) || \
```

Comment:
Per the reference shared in https://github.com/kokkos/kokkos/pull/8529#issuecomment-3378672173
I expect that with Clang it will work for CUDA >= 11.7

Reply:
The release notes clearly state it was added in 20 https://releases.llvm.org/20.1.0/tools/clang/docs/ReleaseNotes.html#cuda-support

Reply:
Clang 16.0.6 with CUDA 12.6

Reply:
18.1 gives the same warning.